src/main/java/org/cbioportal/persistence/mybatisclickhouse/StudyViewMapper.java

```diff
@@ -13,6 +14,10 @@
 public interface StudyViewMapper {
     List<Sample> getFilteredSamples(StudyViewFilter studyViewFilter, CategorizedClinicalDataCountFilter categorizedClinicalDataCountFilter, boolean applyPatientIdFilters);
 
+    List<AlterationCountByGene> getMutatedGenes(StudyViewFilter studyViewFilter, CategorizedClinicalDataCountFilter categorizedClinicalDataCountFilter, boolean applyPatientIdFilters);
```

Comment:
What does this map to?
If there is one defined below?

Reply:
weird.  this is a mistake